src/containers/InfiniteList.tsx

```diff
@@ -0,0 +1,26 @@
+import InfiniteScroll from "react-infinite-scroller";
+
+import { PaginationParamsResult } from "../types/mock-api-types";
+export interface InfiniteScrollerListProps {
+  // eslint-disable-next-line
+  showMore(page: number): any;
+  data: PaginationParamsResult;
+  // eslint-disable-next-line
+  itemComponent: (item: any) => JSX.Element;
```

Comment:
tell me pleas why you use '=>'? What is this?

Reply:
Not sure if I can explain this better then official docs.